packages/block-editor/src/components/block-list/index.native.js

```diff
@@ -151,10 +153,15 @@ export class BlockList extends Component {
 			isReadOnly,
 			shouldShowInsertionPointBefore,
 			shouldShowInsertionPointAfter,
+			getBlockContainerStyle,
 		} = this.props;
 
 		return (
-			<ReadableContentView>
+			<ReadableContentView
+				style={
+					getBlockContainerStyle && getBlockContainerStyle( clientId )
+				}
+			>
```

Comment:
pass function which takes `clientId` as arguments. The logic to set proper alignment sits in `Columns` block